src/components/MultiSelect/style.scss

```diff
@@ -9,13 +9,6 @@
   &-trigger {
     width: 2.2rem;
   }
-
-  & &-clear-icon {
```

Comment:
Fixed in Prime v.9.5.0 https://github.com/primefaces/primereact/blob/master/CHANGELOG.md#950-2023-05-24